docs/components/card.md

```diff
@@ -74,6 +74,16 @@ String to be used as a title. Can also be [slotted](#slots).
 <KCard title="Card Title" />
 ```
 
+### titleTag
+
+HTML element you want title to be rendered as. Defaults to `div`.
```

Comment:
suggestion: might be worth documenting the allowed tags

Reply:
Done